utils/sphinxtool.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
```

Comment:
maybe something more explicit for the name, gen_partial_doc.py ?

Reply:
Good point but I dislike underscores in script names: is `docfast.py` OK for you?